graphql/src/store/query.rs

```diff
@@ -141,6 +141,24 @@ fn build_fulltext_filter_from_object(
     )
 }
 
+fn parse_change_block_filter(
+    value: &graph::data::value::Value,
```

Comment:
It's minor, but since we have so many `Value` structs floating around worth to be consistent: most of the code that deals with `graph::data::value::Value` actually does a `use graph::prelude::r` and then calls it `r::Value` (the `r` is because we call values from `graphql_parser` `q::Value` and `r` is the next letter ;) )